third_party/bazel/gpus/local_config_cuda.BUILD

```diff
@@ -0,0 +1,60 @@
+load("@local_config_cuda//cuda:build_defs.bzl", "enable_cuda_flag")
```

Comment:
Nit: I think it is better to reuse those cuda bazel local_repository configurations from tf_community. Maintaining those configurations ourselves is not a good idea.

Reply:
Good idea, trying it now.

Reply:
It's changed to use tf_commuinity's gpu logics, these codes are removed.

Reply:
looks good ~ 👍